media-libs/dav1d/dav1d-0.5.2.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="A new AV1 cross-platform decoder"
+DESCRIPTION="
+dav1d is a new AV1 cross-platform decoder, open-source, and focused on speed and correctness. \
+It supports all features from AV1, including all subsampling and bit-depth parameters."
+HOMEPAGE="https://code.videolan.org/videolan/dav1d"
+COPYRIGHT="2018-2019, VideoLAN and dav1d authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://code.videolan.org/videolan/dav1d/-/archive/$portVersion/dav1d-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1780199d71e62a82ca1bfca2d71da5c599c7e1edc68c72f6fc3c0c25ce1880b7"
+SOURCE_DIR="dav1d-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	dav1d = $portVersion
+	lib:libdav1d
+	cmd:dav1d
+	"
+REQUIRES="
+	haiku
+	"
+
+PROVIDES_devel="
+	dav1d_devel = $portVersion
+	devel:libdav1d
+	"
+REQUIRES_devel="
+	dav1d == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:meson >= 0.47
+	cmd:ninja
+	cmd:doxygen
+	cmd:nasm >= 2.13.02
+	"
+
+BUILD()
+{
+	mkdir build && cd build
+
+	meson ..
+
+	ninja $jobArgs
+
+	doxygen doc/Doxyfile
+}
+
+INSTALL()
+{
+	cd build
```

Comment:
Did you try "ninja -C build install" ?

Reply:
`ninja install` works, but puts everything in the wrong place.
It installs to the default path in Linux `usr\`.